.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: Amplify UI Component Issue?
+    url: https://github.com/aws-amplify/amplify-ui/issues
```

Since the customer is starting with https://github.com/aws-amplify/amplify-js/issues/new/choose, should we also point them to our chooser?

```suggestion
    url: https://github.com/aws-amplify/amplify-ui/issues/new/choose
```

Today, it still goes to https://github.com/aws-amplify/amplify-ui/issues/new because we don't have any templates (which is fine), but will eventually be our own chooser?